### PR TITLE
LPS-116577 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LPS-116577
ExperienceCreateExperience#EditInlineTextExperience

### CODE REVIEW

**Comment by manoelcyreno** [comment]
ci:test:sf

**Comment by manoelcyreno** [comment]
ci:test:tango

**Comment by manoelcyreno** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 1]
```diff
@@ -74,4 +74,44 @@ definition {
 			value1 = "Banner Title Example");
 	}
 
+	@description = "Validate inline text changes between experiences"
+	@priority = "5"
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:78]: please add here the relevant flag.
`property portal.acceptance = "true";`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 2]
```diff
@@ -74,4 +74,44 @@ definition {
...
+	test EditInlineTextExperience {
+
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:80]: please add here the relevant flag.
property portal.acceptance = "true";

**Comment by nkkwon** [comment]
ci:forward

**Comment by nkkwon** [comment]
ci:stop

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:forward

**Pull request closed without merging**